device-types/Cisco/WS-C2960XR-24TD-I.yaml

```diff
@@ -75,5 +75,7 @@ console-ports:
 module-bays:
   - name: PSU0
     position: '0'
+    type: iec-60320-c14
   - name: PSU1
     position: '1'
+    type: iec-60320-c14
```

Comment:
You dont define PSU type in the module bay, that is part of the module-type file